confluent_kafka/schema_registry/schema_registry_client.py

```diff
@@ -403,11 +498,11 @@ def get_compatibility(self, subject_name=None):
 
         """  # noqa: E501
         if subject_name is not None:
-            url = url_formatter(CONFIG, subject_name)
+            url = 'config/{}'.format(subject_name)
         else:
-            url = url_formatter(CONFIGS)
+            url = 'config'
 
-        return self._rest_client.GET(url, decoder=_CompatibilityJSONDecoder)
+        return self._rest_client.GET(url)
 
 
 class CompatibilityType(object):
```

Comment:
are we sure this is how enums should be implemented? (i don't know, not a python person).

want to note though that the question can be completely avoided here, since you don't need the set and get compatibility methods for the purposes of implementing the serdes - you could just remove the lot.